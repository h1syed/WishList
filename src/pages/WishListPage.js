import React from "react";
// import { Loading, Tabs, Icon } from "element-react";

class WishListPage extends React.Component {
  state = {};

  render() {
    return <div>WishListPage</div>;
  }
}

export default WishListPage;