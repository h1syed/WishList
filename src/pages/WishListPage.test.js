import React from "react";
import { shallow } from "enzyme";
import WishListPage from "./WishListPage";

describe("WishListPage", ()=>{
    const wishlistpage = shallow(<WishListPage />);

    it("renders properly", () => {
        expect(wishlistpage).toMatchSnapshot();
    });

   

    
});