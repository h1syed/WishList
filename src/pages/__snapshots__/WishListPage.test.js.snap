// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WishListPage renders properly 1`] = `ShallowWrapper {}`;
